changelog/README.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 17th March 2025
+* [Reservation](../mews-operations/reservations.md) extended with `booker` property (NOT a breaking change). 
```

Comment:
This is my recommendation for the format of the Changelog, following our new guidelines.
```suggestion
* [Mews: Process group](../mews-operations/reservations.md#process-group):
  * Extended request object with new `booker` property.
```